shared/teams/team/rows/index.tsx

```diff
@@ -64,11 +64,15 @@ const makeRows = (
             })
           }
           rows.push(
-            ...[...details.requests].sort().map(username => ({
-              key: `invites-request:${username}`,
-              type: 'invites-request' as const,
-              username,
-            }))
+            ...[...details.requests].sort().map(req => {
```

Comment:
This `sort` needs an arg now that `requests` isn't strings

Reply:
I removed sort in another branch and didn't merge it, oops. The requests in details are already sorted.